uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1024.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+trait CRM_HRCore_Upgrader_Steps_1024 {
+
+  /**
+   * Remove Social Account Options and reorder de remaining
+   */
+  public function upgrade_1024() {
+    $this->up1024_removeSocialAccountOptions([
+      'Work',
+      'MySpace',
+      'Vine',
+      'Google+',
+      'Snapchat',
+      'Tumblr',
+    ]);
+    $this->up1024_reorderSocialAccountOptions([
+      'LinkedIn',
+      'Twitter',
+      'Facebook',
+    ]);
+
+    return TRUE;
+  }
+
+  /**
+   * Removes Some Social Account Options
+   */
+  private function up1024_removeSocialAccountOptions($accounts) {
+    $socialAccounts = civicrm_api3('OptionValue', 'get', [
+      'option_group_id' => 'website_type',
+      'name' => ['IN' => $accounts],
+    ]);
+
+    $socialAccounts = $socialAccounts['values'];
+    foreach ($socialAccounts as $optionValueId => $optionValue) {
+      civicrm_api3('OptionValue', 'delete', [
```

Comment:
This implementation only deletes without checking if the social account options are in use.